SofaKernel/modules/Sofa.Topology/src/sofa/topology/Topology.h

```diff
@@ -0,0 +1,40 @@
+/******************************************************************************
+*                 SOFA, Simulation Open-Framework Architecture                *
+*                    (c) 2006 INRIA, USTL, UJF, CNRS, MGH                     *
+*                                                                             *
+* This program is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU Lesser General Public License as published by    *
+* the Free Software Foundation; either version 2.1 of the License, or (at     *
+* your option) any later version.                                             *
+*                                                                             *
+* This program is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License *
+* for more details.                                                           *
+*                                                                             *
+* You should have received a copy of the GNU Lesser General Public License    *
+* along with this program. If not, see <http://www.gnu.org/licenses/>.        *
+*******************************************************************************
+* Authors: The SOFA Team and external contributors (see Authors.txt)          *
+*                                                                             *
+* Contact information: contact@sofa-framework.org                             *
+******************************************************************************/
+#pragma once
+
+#include <sofa/topology/config.h>
+
+#include <sofa/geometry/ElementType.h>
+
+#include <sofa/topology/Point.h>
+#include <sofa/topology/Edge.h>
+#include <sofa/topology/Triangle.h>
+#include <sofa/topology/Quad.h>
+#include <sofa/topology/Pentahedron.h>
+#include <sofa/topology/Tetrahedron.h>
+#include <sofa/topology/Pyramid.h>
+#include <sofa/topology/Hexahedron.h>
+
+namespace sofa::topology
+{
+
+}
```

Comment:
empty namespace needed?

Reply:
just in case we needed to put something inside, but I will delete it